mteb/abstasks/TaskMetadata.py

```diff
@@ -87,7 +89,35 @@
 
 
 class TaskMetadata(BaseModel):
-    # Meta: We can annotate the requirements here, and then link to it in the docs. This would move the documentation closer to the code, which I think is a good idea.
+    """
+    Metadata for a task.
+
+    Args:
```

Comment:
This ends up duplicating a lot of information that is already present in the type-system. Personally, I'd prefer inlining the types, so we get the information in the auto-complete in an IDE. That'd decrease the maintenance burden, and ensure alignment between annotations and actual data.

That said, we hope this will change very little, so if there's a strong preference for docstring, that's fine by me.